features/facebookImport/src/model/analyses/ministories/advertising-value-analysis.js

```diff
@@ -28,14 +31,25 @@ export default class AdvertisingValueAnalysis extends RootAnalysis {
         }
         this._randomAdInterests = [...randomAdInterests];
         this._numberInterests = numberInterests;
+        this._displayData = adInterests.sort((a, b) => {
+            return a.toLowerCase().localeCompare(b.toLowerCase());
```

Comment:
But why do you need this? First, will interests be in the same script as the device locale or in English? Second, why do you need to go lowercase? Take into account that the transformation will be made *in every comparison*. So if it does not matter how they are eventually printed, it might be better to first convert to lowercase, and then sort.